docs/intro_about.rst

```diff
@@ -56,6 +56,72 @@ matrices, rotation matrices, symmetric matrices, or permutation matrices. When :
++--------------------------------------------------------------------------------------------------------------------+------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| :ref:`Symmetric <symmetric>` :cite:`symmetric:1,escalante1998dykstra,peng2008m`                                    | :math:`\mathbf{I}`           | :math:`\mathbf{X}`   | :math:`\mathbf{X} = \mathbf{X}^\dagger`                                                                                                                                                                                                                         |
++--------------------------------------------------------------------------------------------------------------------+------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| :ref:`Permutation <permutation>` :cite:`zadeh2013molecular`                                                        | :math:`\mathbf{I}`           | :math:`\mathbf{P}`   | :math:`\begin{cases} [\mathbf{P}]_{ij} \in \{0, 1\} \\ \sum_{i=1}^n [\mathbf{P}]_{ij} = \sum_{j=1}^n [\mathbf{P}]_{ij} = 1 \\ \end{cases}`                                                                                                                      |
++--------------------------------------------------------------------------------------------------------------------+------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| :ref:`Two-sided Orthogonal <orthogonal>` :cite:`two_sided_ortho:1`                                                 | :math:`\mathbf{Q}_1`         | :math:`\mathbf{Q}_2` | :math:`\begin{cases} \mathbf{Q}_1^{-1} = \mathbf{Q}_1^\dagger \\ \mathbf{Q}_2^{-1} = \mathbf{Q}_2^\dagger \\ \end{cases}`                                                                                                                                       |
```

Comment:
Change `:math:`\mathbf{Q}_1` to `:math:`\mathbf{Q}_1^\dagger` (I changed the table in the paper a little while ago, it makes things simpler to keep the dagger for U1)

Reply:
It's fixed in 860b02b.